apps/datahub/src/app/record/header-record/header-record.component.ts

```diff
@@ -34,7 +34,14 @@ export class HeaderRecordComponent {
   )
 
   get lastUpdate() {
-    return this.metadata.recordUpdated.toLocaleString().split(',')[0]
+    const date = this.metadata.recordUpdated
+    return (
+      ('0' + date.getDate()).slice(-2) +
+      '.' +
+      ('0' + (date.getMonth() + 1)).slice(-2) +
+      '.' +
+      date.getFullYear()
+    )
```

Comment:
I'd try to prevent a custom function here (that will make localization much more complicated as well).
How about a `return this.metadata.recordUpdated.toLocaleDateString(this.translateService.currentLang)`?
This will not display the dots from the mockup, but slashes are French standard.

Reply:
Okay! I knew it wasn't the best, but I didn't find anything better to have the dots. If we ditch the dots, then yes, your suggestion is much better!